src/components/electron/window-controls.tsx

```
@@ -0,0 +1,157 @@
+  // Don't render in browser
+  if (!isElectron()) {
+    return null;
+  }
+
+  // Don't render on macOS (uses native traffic light buttons)
+  if (platform === 'macos') {
+    return null;
+  }
+
+  // Subscribe to window state changes
+  useEffect(() => {
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

Hooks should not be called after an early return.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)